routes/fractie.ts

```diff
@@ -0,0 +1,30 @@
+import Router from 'express-promise-router';
+
+import { Request, Response } from 'express';
+import { fractieUsecase } from '../controllers/fractie';
+import { STATUS_CODE } from '../util/constants';
+
+export const fractiesRouter = Router();
+
+fractiesRouter.post(
```

Comment:
AS we now added the bestuursperiode when looking for  the onafhankelijke fractie we could make the call that it will always return an onafhankelijke fractie  (when not found create one directly). Leaving it in